server/src/test/java/org/elasticsearch/index/store/StoreTests.java

```diff
@@ -859,7 +860,7 @@ public void testStreamStoreFilesMetaData() throws Exception {
         Store.MetadataSnapshot metadataSnapshot = createMetaDataSnapshot();
         TransportNodesListShardStoreMetaData.StoreFilesMetaData outStoreFileMetaData =
             new TransportNodesListShardStoreMetaData.StoreFilesMetaData(new ShardId("test", "_na_", 0),
-                metadataSnapshot);
+                metadataSnapshot, Collections.emptyList());
```

Comment:
I think we should add something random here and validate that it is preserved when streaming.